pub/index.html

```diff
@@ -4,40 +4,154 @@
+            <aside class="sidebar">
+                <figure class="aside-image">
+                    <img src="http://lorempixel.com/output/cats-q-c-100-100-7.jpg">
+                </figure>
+                <h4>Cattie Cat</h4>
+                <nav class="aside-nav">
+                    <a href="#">Twitter</a>
+                    <a href="#">Facebook</a>
+                    <a href="#">Instagram</a>
+                </nav>
+                <figure class="ad-area">
+                    <img src="https://placekitten.com/g/600/600">
+                    <figcaption class="ad-text">
+                        <a href="#">About This Ad</a>
+                </figcaption>
+                </figure>
+                <div class="hidden-mobile">
+                    <h5>Sidebar Section Heading</h5>
+                    <p>Find something else more interesting. Roll on the floor purring your whiskers off swat at dog. Sun bathe play riveting piece on synthesizer keyboard why must they do that, so rub face on everything hunt anything that moves hiss at vacuum cleaner.</p>
+                </div>
+            </aside>
+        </div> 
+        <section class="bottom-content">
+            <h2>Most Popular Posts</h2>
+            <ul class="image-container">
+                <li class="post">
+                    <figure class="image">
+                        <img src="https://placekitten.com/g/600/600">
+                        <figcaption class="image-title">Title</figcaption>
+                    </figure>
                 </li>
-            </ol>
-        </div>    
+                <li class="post">
+                    <figure class="image">
+                        <img src="https://placekitten.com/g/600/600">
+                        <figcaption class="image-title">Title</figcaption>
+                    </figure>
+                </li>
+                <li class="post">
+                    <figure class="image">
+                        <img src="https://placekitten.com/g/600/600">
+                        <figcaption class="image-title">Title</figcaption>
+                    </figure>
+                </li>
+                <li class="post">
+                    <figure class="image">
+                        <img src="https://placekitten.com/g/600/600">
+                        <figcaption class="image-title">Title</figcaption>
+                    </figure>
+                </li>
+            </ul>
+        </section>
+        <footer class="page-footer">
+            <nav class="nav-footer">
+                <a href="#">Nav Item 1  |</a>
+                <a href="#">Nav Item 2  |</a>
+                <a href="#">Nav Item 3  |</a>
+                <a href="#">Nav Item 4  |</a>
+                <a href="#">Nav Item 5</a>
+            </nav>
+            <div class="copyright-content">
+                <img src="img/tiy-logo.png">
+                <p>The Ironyard Copyright 2013 All Rights Reserved</p>
+            </div>
```

Comment:
Good choice putting the footer nav _above_ the copyright block in your HTML. Even though it appears to the _right_ in the full-sized view, the nav appears _before_ the copyright in mobile.